sdk/ml/azure-ai-ml/azure/ai/ml/_schema/job/input_output_entry.py

```diff
@@ -59,6 +59,16 @@ def generate_path_property(azureml_type):
     )
 
 
+def generate_path_on_compute_property(azureml_type):
+    return UnionField(
+        [
+            ArmVersionedStr(azureml_type=azureml_type),
```

Comment:
Not sure if these patterns are correct for how Users will specify `path_on_compute`, generally it's expressed as a string which is a local path like: `/tmp/my_custom_path`.

Reply:
As it is always a local path, so we should omit 'file:' ?

Reply:
thank you for your suggestions, you are right and I have fixed it